packages/utils/conversion.ts

```diff
@@ -172,8 +172,36 @@ export const durationToSeconds = (blocksPerYear: number, duration: Duration) =>
     ? convertBlocksToSeconds(blocksPerYear, duration.height)
     : duration.time
 
-// Use Stargaze's IPFS gateway.
+// Convert IPFS protocol URL to HTTPS protocol URL using IPFS gateway.
 export const transformIpfsUrlToHttpsIfNecessary = (ipfsUrl: string) =>
   ipfsUrl.startsWith('ipfs://')
     ? IPFS_GATEWAY_TEMPLATE.replace('PATH', ipfsUrl.replace('ipfs://', ''))
     : ipfsUrl
+
+// Normalize image URLs by ensuring they are from a valid IPFS provider or not
```

Comment:
Is "normalize" really what we mean here? Or is it something like `toValidIPFSUrlWithProvider`?

Reply:
True, renamed and added more comments!